poc_mobile_verifier/src/cli/reward_from_db.rs

```diff
@@ -32,18 +31,24 @@ impl Cmd {
         let expected_rewards = get_scheduled_tokens(epoch.start, epoch.end - epoch.start)
             .expect("Couldn't get expected rewards");
 
-        let follower = settings.follower.connect_follower()?;
+        let mut follower = settings.follower.connect_follower()?;
         let pool = settings.database.connect(10).await?;
 
         let heartbeats = Heartbeats::validated(&pool, epoch.start).await?;
         let speedtests = SpeedtestAverages::validated(&pool, epoch.end).await?;
-        let rewards =
-            SubnetworkRewards::from_epoch(follower, &epoch, heartbeats, speedtests.clone()).await?;
+        let owner_shares = follower
+            .owner_shares(heartbeats, speedtests.clone())
```

Comment:
see below; i think implementing this method against the `OwnerResolver` makes this looks like the follower is doing the calculations